native/packages/react-native-bpk-component-banner-alert/src/customPropTypes.js

```diff
@@ -18,6 +18,8 @@ import { isString } from 'util';
 * limitations under the License.
 */
 
+/* @flow */
+
 export const dismissablePropType = (props, propName, componentName) => {
```

Comment:
Shouldn't this been in common-types too? Given that all the other prop types are there

Reply:
There are no custom prop types inside `common-types.js`. I mean the file separation still makes sense...

Reply:
Yeah but all the prop types for the banner alert are in there, it's confusing at the very least